scripts/averaging/create_TEM_files.py

```diff
@@ -146,8 +127,6 @@ def create_TEM_files(adf):
         tem_fil = tem_base_loc / f'{base_name}.TEMdiag.nc'
```

Comment:
@nusbaume Since we don't have a dedicated obs location for saved files, the best I could come up with quickly was just to save the obs TEM file to the same location as the first test case TEM file location. Let me know if you have thoughts for or against this for now.